app/services/conviction_decision_tree.rb

```diff
@@ -47,13 +47,13 @@ def destination
   def after_conviction_subtype
     return edit(:conviction_bail)   if conviction_subtype.bailable_offense? && !step_name.eql?(:bypass_bail_conviction_subtype)
     return edit(:compensation_paid) if conviction_subtype.compensation?
-    return after_adult_motoring     if conviction_subtype.parent.inquiry.adult_motoring? || conviction_subtype.parent.inquiry.youth_motoring?
+    return after_adult_motoring     if conviction_subtype.parent.motoring?
```

Comment:
This can be just `conviction_type.motoring?`

Also, just to be coherent, what if we change the method name from `#after_adult_motoring` to something more generic, kind of `#after_motoring_conviction`?